### PR TITLE
Fix the bundles used in e2e tests

### PR DESCRIPTION
**Pull request by hbenl**
- added original sources and webpack configs for our test bundles
- updated the bundles
- the test runner now uploads the example recordings directly when using firefox (instead of recording to a file and uploading that afterwards) so that they can contain sourcemaps (see RecordReplay/gecko-dev#435)
- chromium tests are currently broken, see RecordReplay/chromium#25
- the example recordings are now added correctly to `test/example-recordings.json`
- reenabled 9 tests
- the breakpoints-07 test contains a navigation, after that navigation playwright seems semi-broken and the test doesn't continue - perhaps this could be fixed by rewriting the tests to not run in the browser anymore (see #5137) so that the navigation is triggered using the playwright API
- the object_preview-05 test fails due to a scope mapping issue in the backend (RecordReplay/backend#4483)
- the react_devtools test doesn't work with an example created with playwright because that browser doesn't support react devtools yet - you can run that test by adding `"cra/dist/index.html": "acbf7e7e-9ad7-44f1-a5bd-70aa8b6c0fea"` to `test/example-recordings.json` (that recording was made manually with our standard gecko browser)

### CODE REVIEW

File: src/test/harness.js [hunk 1]
```diff
@@ -138,14 +138,14 @@ async function selectSource(url) {
 }
 
 async function addLogpoint(url, line) {
-  const bpCount = dbgSelectors.getBreakpointCount();
+  const bpCount = dbgSelectors.getLogpointCount();
```
Comment (abstractalgo) [inline src/test/harness.js:141]: `lpCount` then, not `bpCount` 😄

File: test/src/getExample.js [hunk 2]
```diff
@@ -51,14 +87,25 @@ async function upload(state, example) {
...
+  state.exampleRecordings = { ...state.exampleRecordings, [example]: recordingId };
+  fs.writeFileSync(
+    "./test/example-recordings.json",
+    JSON.stringify(state.exampleRecordings, null, 2)
+  );
 }
 
 async function recordExample(state, example, target) {
+  const exampleUrl = `${state.testingServer}/test/examples/${example}`;
   const browser = target == "gecko" ? "firefox" : "chromium";
```
Comment (abstractalgo) [inline test/src/getExample.js:99]: `===`

File: package.json [hunk 3]
```diff
@@ -149,7 +154,9 @@
     "style-loader": "^3.2.1",
     "ts-node": "^10.0.0",
     "tsconfig-paths": "^3.9.0",
-    "typescript": "^4.5.4"
+    "typescript": "^4.5.4",
+    "webpack": "^5.68.0",
+    "webpack-cli": "^4.9.2"
```
Comment (abstractalgo) [inline package.json:159]: We could create a `package.json` specifically for tests, instead of pollute a global one.

**Comment by hbenl** [comment]
Superseded by #5275

**Pull request closed without merging**